python/cudf/cudf/_lib/cpp/io/parquet.pxd

```diff
@@ -76,7 +76,7 @@ cdef extern from "cudf/io/parquet.hpp" namespace "cudf::io" nogil:
             vector[cudf_io_types.partition_info] partitions
```

Comment:
Line 68 (which provide the forward-decl for `parquet_writer_options.get_metadata()` also wants changed I think.